pkg/util/csiconfig.go

```diff
@@ -23,6 +23,12 @@ import (
 	"strings"
 )
 
+// clusterInfo strongly typed JSON spec for the above JSON structure
+type clusterInfo struct {
+	ClusterID string   `json:"clusterID"`
```

Comment:
Comments should probably also be added inside this struct:

```golang
// clusterInfo blablabla
type clusterInfo struct {
        // ClusterID is a unique ..... whatever it is
        ClusterID ....
...
}
```

Reply:
Thanks, will add them.